### DISCUSSION TITLE
Live Preview with Server: multiple sites

### DISCUSSION THREAD

**Discussion by scrollable-sidebar** [phcode-dev/phoenix#1775] [Q&A] [answered]
I use Laragon (not sure that matters) and am working on multiple sites. My folder structure looks likes this:

www\site1
www\site2
www\site3

If I follow the instructions to edit my httpd.conf file like the below, it works for that one site:

DocumentRoot: "D:/Laragon/www/site1"
<Directory "D:/Laragon/www/site1">

But I'd like to have my Document Root look like this:

DocumentRoot: "D:/Laragon/www"
<Directory "D:/Laragon/www">

...and then enter "site1", or "site2" etc in Pheonix's Live Preview Settings. Something like this:

![Screenshot 2024-07-24 213743](https://github.com/user-attachments/assets/cc2bdaf1-c931-4ebf-b118-ba992216300d)

So far I've been unsuccessful - is it possible?

**Comment by abose** [comment, marked as answer]
If the site url is something like `http://localhost/site1` , `http://localhost/site2` , etc. then 
* open the folder `www` in phoenix code.
* give the url in server settings as `http://localhost`
* serving folder in project should be just `/`

This should work.  Please see https://docs.phcode.dev/docs/Features/live-preview-settings/

**Reply by scrollable-sidebar** [reply]
Thank you for the quick solution! I've now got it working but I can see that I have various live preview issues relative to Brackets which I'll try to figure out.

**Reply by abose** [reply]
Please do drop any issues you find relative to Brackets here or as new issues. That will help us fix issues for users in brackets feature complete and migration.
